src/create_gmad.cpp

```diff
@@ -3,6 +3,7 @@
 #include "AddonWhiteList.h"
 #include "AddonFormat.h"
 #include "Addon_Json.h"
+#include "AddonBuffer.h"
```

Comment:
This whole hack is not necessary. There's no 500MB limit with the Bootil's AutoBuffer in GMod.

Reply:
I will change it to use Bootil's AutoBuffer